<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadWindow</class>
 <widget class="QMainWindow" name="DownloadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Progress</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #0F0F12;
border: 0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #1A1A1F;</string>
   </property>
   <widget class="QFrame" name="titleFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1F;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="windowTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #E5E7EB;</string>
     </property>
     <property name="text">
      <string>Download Progress</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Font Awesome 7 Free</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: #E5E7EB;
	border: 0;
}

QPushButton:hover {
	background: #EF4444;
}</string>
     </property>
     <property name="text">
      <string>ÔÄç</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>600</width>
      <height>370</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0F0F12;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="filenameEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>560</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
        background: #1A1A1F;
        color: #FF4D6D;
        border: 1px solid #27272F;
        border-radius: 5px;
        padding: 0 5px;
      }
      QLineEdit:focus {
        border: 1px solid #FF4D6D;
      }</string>
     </property>
     <property name="placeholderText">
      <string>Filename</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>560</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
        background: #1A1A1F;
        color: #E5E7EB;
        border: 1px solid #27272F;
        border-radius: 5px;
        padding: 0 5px;
      }
      QLineEdit:focus {
        border: 1px solid #FF4D6D;
      }</string>
     </property>
     <property name="placeholderText">
      <string>Download Path</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startDownloadButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>560</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background: #FF4D6D;
        color: #1A1A1F;
        border-radius: 5px;
      }
      QPushButton:hover {
        background: #FF1F4B;
      }
    </string>
     </property>
     <property name="text">
      <string>Start Download</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>560</width>
       <height>10</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	background-color: #1A1A1F;
	border-radius: 5px;
    border: none;
}

QProgressBar::chunk {
	background-color: #FF4D6D;
	border-radius: 5px;
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="logOutput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>560</width>
       <height>200</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #1A1A1F;
color: #E5E7EB;
border-radius: 5px;
padding: 5px;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
